.changeset/plenty-bugs-push.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
question: nitpicking, but should this be a minor? Introduces a couple of minor breaking changes I believe

Reply:
true, the format of the linkableKeys has changed.

Reply:
I don't think it is an issue, I doubt any body has created links yet and we have migrated all of them.

Reply:
is the changeset filename a premonition? 😆